### DISCUSSION TITLE
What "Repetitions per condition" mean?

### DISCUSSION THREAD

**Discussion by karellopez** [cvnlab/GLMsingle#89] [Q&A]
Hi,
I am trying to use the tool. But I do not clearly understand what "Repetitions per condition"  mean. Is it referring to the number of trials within the condition?

Best regards,
Karel

**Comment by kendrickkay** [comment]
Yes.  A "condition" is like a specific experimental manipulation that you design, and "trial" refers to the specific instances of a given condition that occur over the course of an experiment.  "Repetitions" refers to the number of trials that you have for a given condition.
